code/modules/SCPmodular/scps/euclid/SCP-294.dm

```diff
@@ -0,0 +1,72 @@
+GLOBAL_LIST_EMPTY(scp294_reagents)
+
+/obj/machinery/scp294
+	name = "SCP-294"
+	desc = "A standard coffee vending machine."
+	icon = 'code/modules/SCPmodular/spcicon/scpobj/scp294.dmi'
```

Comment:
Move all icons to be under icons/scp

Under no circumstances should icons be in the modules directory